node/pkg/guardiansigner/guardiansigner.go

```diff
@@ -0,0 +1,66 @@
+package guardiansigner
+
+import (
+	"crypto/ecdsa"
+	"fmt"
+	"strings"
+)
+
+// The types of guardian signers that are supported
+type SignerType int
+
+const (
+	InvalidSignerType SignerType = iota
+	FileSignerType
+)
+
+// GuardianSigner interface
+type GuardianSigner interface {
+	// TODO: document that the signer implementations assume they are recieving a keccack256 hash
```

Comment:
Not sure if you want to leave this one in there or not.

Reply:
I was meaning to document the interface. Will do that. Thanks for pointing this out.